### PR TITLE
Content for PostgreSQL CDC connector for Aiven Flink

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Content for PostgreSQL CDC connector for Aiven for Apache Flink service. Include info on how to use the connector in the Flink table.

### CODE REVIEW

File: docs/products/flink/howto/pg-cdc-connector.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+When using Aiven for Apache Flink®, you can seamlessly leverage the power of the PostgreSQL CDC Connector to stream and process real-time data changes from PostgreSQL databases. Integrated with the Debezium engine, the CDC connector captures changes at the granular level of each table within an event stream. 
+
+This article provides you with information on how to create a PostgreSQL CDC connector-based Apache Flink® table. 
+
+Prerequisites
+--------------
+Before setting up the PostgreSQL CDC Connector with Aiven for Apache Flink, ensure that you have the following prerequisites in place:
+
+* Running Aiven for Apache Flink® Service
+* Running Aiven for Apache Kafka® Service
```
Comment (HelenMel) [inline docs/products/flink/howto/pg-cdc-connector.rst:15]: We might say here something like (or any other Sink of your choice)

File: docs/products/flink/howto/pg-cdc-connector.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* ``Port``: The port number on which the PostgreSQL server is listening for incoming connections.
+* ``Database name``: The name of the source database within the PostgreSQL server.
+* ``Username``: The username used to authenticate and access the PostgreSQL database.
+* ``Password``: The password associated with the provided username for authentication.
+* ``Schema name``: The schema name where the source table is located within the database.
+* ``Table name``: The name of the source table from which you want to capture data changes.
+* ``Decoding plugin name``: The decoding plugin name to use for capturing the changes. For PostgreSQL CDC, set it as ``pgoutput``.
+
+.. important:: 
+    To create a PostgreSQL CDC source connector in Aiven for Apache Flink with Aiven for PostgreSQL using the pgoutput plugin, you need to have superuser privileges.
```
Comment (HelenMel) [inline docs/products/flink/howto/pg-cdc-connector.rst:31]: Maybe add here a link how to enable it from our kafka connector blog?

File: docs/products/flink/howto/pg-cdc-connector.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* ``password``: The password for the provided username.
+* ``schema-name``: The name of the schema where the source table is located, which is set to ``public`` in the example.
+* ``table-name``: The name of the source table to be captured by the CDC connector, which is set to ``test-1`` in the example.
+* ``port``: The port number of the PostgreSQL database server.
+* ``database-name``: The name of the database where the source table resides, which is set to ``defaultdb`` in the example.
+* ``decoding.plugin.name``: The decoding plugin to be used by the CDC connector, which is set to ``pgoutput`` in the example.
+
+8. Select **Next** to add the sink table, and then select **Add your first sink table**. Select *Aiven for Apache Kafka®* as the integrated service from the drop-down list.
+9.  In the **Table SQL** section, input the SQL statement for creating the sink table where the PostgreSQL CDC connector will send the data. ßßSelect **Add table**.
+10. In the **Create statement** section, write the SQL schema that defines the fields retrieved from the PostgreSQL® table and any additional transformations.
```
Comment (HelenMel) [inline docs/products/flink/howto/pg-cdc-connector.rst:80]: Step 11 deploy job?

Comment (harshini-rangaswamy) [reply docs/products/flink/howto/pg-cdc-connector.rst:80]: added this step.

File: docs/products/flink/howto/pg-cdc-connector.rst (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/pg-cdc-connector.rst]: Included a troubleshooting section and cross-linked to this note.

**Review by HelenMel — APPROVED** [review]

**Pull request merged**